### ISSUE TITLE
Outdated packages

### ISSUE THREAD

**Issue by git-bruh** [kisslinux/community#1681]
@noocsharp
- [ ] autoconf 2.70 -> 2.71

@onodera-punpun
- [ ] blender 2.91.0 -> 2.91.2

@xuxiaodong
- [ ] bottom 0.5.6 -> 0.5.7

@dylanaraps
- [ ] btrfs-progs 5.10 -> 5.10.1

@onodera-punpun
- [ ] dolphin-emu 5.0.12247 -> 5.0.13495, 5.0, 5

@git-bruh
- [ ] duf 0.5.0 -> 0.6.0

@monarrk
- [ ] elvish 0.14.1 -> 0.15.0

@abbbi
- [ ] entr 4.6 -> 4.7

@Vouivre
- [ ] exiftool 12.16 -> 12.17

@dilyn-corner
- [ ] extra-cmake-modules 5.78.0 -> 5.79.0

@mcpcpc
- [ ] fldigi 4.1.17 -> 4.1.18

@trbednarzyk
- [ ] fuse 3.10.1 -> 3.10.2

@mmatongo
- [ ] gitlab-glab 1.13.1 -> 1.14.0

@noocsharp
- [ ] go 1.15.7 -> 1.16rc1, 1.15.8

@onodera-punpun
- [ ] golangci-lint-bin 1.35.2 -> 1.36.0

@trbednarzyk
- [ ] graphviz 2.44.1 -> 2.46.0

@Ominitay
- [ ] i3-gaps 4.19 -> 4.19.1

@Ominitay
- [ ] i3 4.19 -> 4.19.1

@konimex
- [ ] iana-etc 20200910 -> 20210202

@dylanaraps
- [ ] imagemagick 7.0.10-60 -> 7.0.10-62, 7.0.10.62

@mmatongo
- [ ] intel-media-driver 20.4.5 -> 21.1.1

@magenbluten
- [ ] iputils 20200821 -> 20210202

@onodera-punpun
- [ ] ispc 1.2.4 -> 1.15.0

@Vouivre
- [ ] lcms 2.11 -> 2.12

@dylanaraps
- [ ] libburn 1.5.2.pl01 -> 1.5.4

@mmatongo
- [ ] libcap 2.47 -> 2.48

@eudaldgr
- [ ] libisoburn 1.5.2 -> 1.5.4

@mcpcpc
- [ ] libsamplerate 0.1.9 -> 0.2.1

@mcpcpc
- [ ] libsndfile 1.0.30 -> 1.0.31

@konimex
- [ ] lvm2 2.03.10 -> 2.03.11

@Himmalerin
- [ ] mercurial 5.7rc0 -> 5.7

@DevonMorris
- [ ] mit-scheme 10.1.11 -> 11.1

@magenbluten
- [ ] nawk-git 20180827 -> 20121220

@onodera-punpun
- [ ] opencolorio 1.1.1 -> 2.0.0

@onodera-punpun
- [ ] openimageio 2.2.10.1 -> 2.2.11.1

@magenbluten
- [ ] openvpn 2.4.9 -> 2.5, 2.5.0

@trbednarzyk
- [ ] parted 3.3 -> 3.4

@magenbluten
- [ ] pkcs11-helper 1.26.0 -> 1.27, 1.27.0

@onodera-punpun
- [ ] ppp 2.4.8 -> 2.4.9

@dylanaraps
- [ ] python2 2.7.18 -> 3.10.0a5, 3.10.0~a5, 3.10.0_alpha5, 3.9.1

@xuxiaodong
- [ ] rclone 1.53.4 -> 1.54.0

@mcpcpc
- [ ] rtl-sdr 0.6.0 -> 0.8.0

@aicsx
- [ ] scrot 0.8 -> 1.5

@aaronNGi
- [ ] socat 1.7.4.0 -> 2.0.0

@Bridouz
- [ ] syncthing 1.12.1 -> 1.13.1

@ioraff
- [ ] tcl 8.6.10 -> 8.7.a3, 8.6.11

@aaronNGi
- [ ] terminus-font 4.49 -> 4.49.1

@ioraff
- [ ] tk 8.6.10 -> 8.7.a3, 8.6.11.1

@mmatongo
- [ ] tor 0.4.4.6 -> 0.4.5.5, 0.4.4.7

@ioraff
- [ ] urlview 0.9-21 -> 0.9

Author: Cliford Sab <clifordsab@disroot.org>
- [ ] uthash 2.1.0 -> 2.2.0

@trbednarzyk
- [ ] w3m 0.5.3 -> 0.5.3pl38

@djt3
- [ ] xwallpaper 0.6.5 -> 0.6.6

**Comment by mmatongo** [comment]
Oh my

**Comment by mmatongo** [comment]
I've deliberately refrained because I know that a merge isn't possible in the absence of @dylanaraps

**Comment by eudaldgr** [comment]
I don't maintain packages here anymore, repology have mistaken.

**Comment by dilyn-corner** [comment]
Qt6 will either not be added or will be maintained alongside qt5 until qt5 apps are confirmed working with qt6. 

I don't even think qt6-webengine exists right now? 

nss and ECM will be updated later. I've fallen lax on it while I work on personal projects; afaik nobody but me really hard them anyways 🙃

**Comment by git-bruh** [comment]
List was mostly generated with `kiss-outdated`, will have to clean out false positives and stuff.

**Comment by dilyn-corner** [comment]
For sure, it's just worth noting somewhere publicly available (as opposed to bring trapped inside my mind). 

To my knowledge, it's just graphicz which requires qt (that I don't maintain), so we'd be waiting on the confirmation there.

**Comment by acheam0** [comment]
@mmatongo [kiss-community/repo-community](https://github.com/kiss-community/repo-community) is maintaining and merging PRs, so feel free to make one there, and if/when Dylan comes back we can merge back into kisslinux/community.

**Comment by mmatongo** [comment]
> @mmatongo [kiss-community/repo-community](https://github.com/kiss-community/repo-community) is maintaining and merging PRs, so feel free to make one there, and if/when Dylan comes back we can merge back into kisslinux/community.

Ahhh, will make pulls tomorrow.

**Comment by kiedtl** [comment]
@git-bruh As I had explained to Dylan, since I no longer have access to a x86_64 Linux machine, I have temporarily dropped maintainership of my packages until I can use a suitable Linux computer again. Dylan was maintaining my packages in the meantime. Someone else may have to maintain them until Dylan comes back.

(I would submit a PR from the GitHub website that just bumps the version numbers, but I'd prefer to not do that unless it becomes clear that there's no one else willing to maintain them until Dylan reappears.)

**Comment by jedahan** [comment]
I might make a little poll to ask maintainers what they do for packages. Honestly for me I bump version, install, check --version, maybe run the app and play around for like 30 seconds before declaring it 'good enough'.

If its a library, I just rebuild everything that depends on it and do the same thing.

But most of my packages are basic or self-contained.

**Comment by sdsddsd1** [comment]
nettle #1683 
ruby #1684 
@git-bruh : How do you feel about pushing those commits personal by the maintainers to repo-community? I guess it could be less work to just push everything without looking what already has been pushed.

**Comment by git-bruh** [comment]
@sdsddsd1 Do you mean giving write access to maintainers or me manually `git am`-ing the PRs?

**Comment by sdsddsd1** [comment]
No, I mean more like, should I push my updates from `community` to `repo-community` myself, or does this create extra effort to sort things out which have already been pushed.

**Comment by jedahan** [comment]
I think opening PRs against `repo-community` is good practice both for bookkeeping and maybe a new kind of development model.

**Comment by sdsddsd1** [comment]
Answer in irc towards my pr question:
```
... i think it's better that i manually merge PRs instead of maintainers sending PRs to both community repos...
```

**Comment by git-bruh** [comment]
reasoning for not preferring double PRs is that only a few maintainers send PRs to repo-community along with this repo, and others may or may not do so, which means the other PRs will have to be manually merged anyways, so this helps avoid some confusion that might occur

**Comment by a-schaefers** [comment]
@dylanaraps I thought I had adopted out or orphaned all of my packages, but regardless, I am not going to be helping with this. Thank you, and hope all is well with you and the KISS project. :-)

**Comment by Vouivre** [comment]
Sorry, it's not clear for me. I understood the PR must be sent only to one repository. But which one: kiss-community or community ?

**Comment by git-bruh** [comment]
@Vouivre just send one PR to kisslinux/community so that Dylan can merge everything once he is back, currently all PRs will be merged manually on kiss-community

**Comment by Vouivre** [comment]
@git-bruh : thank you! I will also push my changes here.

**Comment by git-bruh** [comment]
Wrote a shitty python script for creating such issues and automatically tagging maintainers: https://github.com/git-bruh/kiss-repo/blob/main/repo/kiss-outdated-issue/files/kiss-outdated-issue

**Comment by mmatongo** [comment]
Made a few pulls but my laptop is struggling as it is on account of me currently compiling `mariadb`, I will do the rest tomorrow.

**Comment by git-bruh** [comment]
updated the list again

**Comment by Vouivre** [comment]
What is the best way to update a package if it's not available on community ? 
I have packaged `exiftool` and it's on repo-community, but not in community. In community I should send a PR for a new package, on repo-community, only the update. 
Probably the best way it's to clone repo-community. What do you think @git-bruh ?

**Comment by git-bruh** [comment]
@Vouivre Just create a single PR with the latest package version on kisslinux/community, no need to fork kiss-community

**Comment by Vouivre** [comment]
Ok, thanks! :+1:

**Comment by artemkobets** [comment]
@git-bruh https://github.com/kisslinux/community/pull/1676
oath-toolkit 2.6.5 hasn't been merged upstream yet, so I updated existing PR with a commit for 2.6.6.
Is this alright with you or would it be easier to track these with a new PR for each version update instead?

**Comment by git-bruh** [comment]
@artemkobets Most people just close their old PR and create a new, updated one which is better because PRs just show as "Opened X days ago" and the date won't update on pushing new commits

**Comment by git-bruh** [comment]
repology now points to new community repo:
https://repology.org/repository/kiss_community

**Comment by ominitay** [comment]
When is it planned for the new repo to be merged into this?

**Comment by git-bruh** [comment]
@Ominitay Dylan's been gone for a few months now and no one else has write access to this repo so... nothing can happen till he's back

**Comment by ominitay** [comment]
Ah right! Perhaps a community repo with multiple people with write access would be a good idea to keep then :)

**Comment by git-bruh** [comment]
@Ominitay Yes, I've invited a couple of maintainers to @kiss-community /repo-community

**Comment by git-bruh** [comment]
PR here to drop outdated packages that have no response from maintainers: https://github.com/kiss-community/repo-community/pull/89

If anyone is interested in adopting a package, they can make a comment on that PR.

**Comment by git-bruh** [comment]
https://github.com/kiss-community/repo-community/issues/97
